<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="toggleDigitalButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Digital</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle12HourModeButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>12 Hour</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="darkModeRadioButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>447</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark Mode</string>
    </property>
   </widget>
   <widget class="ClockFace" name="clockFace">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="lightModeRadioButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>467</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="digitalDisplayLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>31</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>12:44 PM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle24HourModeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>24 Hour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleDashLinesButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dash Lines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleMinuteHandButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Minute Hand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleHourHandButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hour Hand</string>
    </property>
   </widget>
   <widget class="QSlider" name="HourHandSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="minuteHandSlider">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>490</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hour hand slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>510</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minute hand slider</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="blueModeRadioButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="redModeRadioButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClockFace</class>
   <extends>QGraphicsView</extends>
   <header location="global">ClockFace.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
